From dc13361524c1477b2106c7385f2059f9ea111b84 Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Wed, 5 Jul 2023 09:29:54 +0900
Subject: [PATCH] dirmngr: Enable the call of ks_ldap_help_variables when
 USE_LDAP.

* dirmngr/server.c [USE_LDAP] (cmd_ad_query): Conditionalize.

--

Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>

Upstream: https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=commitdiff;h=dc13361524c1477b2106c7385f2059f9ea111b84

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 dirmngr/server.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dirmngr/server.c b/dirmngr/server.c
index 51a149cb2..ee61f63d6 100644
--- a/dirmngr/server.c
+++ b/dirmngr/server.c
@@ -2776,7 +2776,9 @@ cmd_ad_query (assuan_context_t ctx, char *line)
 
   if (opt_help)
     {
+#if USE_LDAP
       ks_ldap_help_variables (ctrl);
+#endif
       err = 0;
       goto leave;
     }
-- 
2.11.0

